apps/testrunner/tasks/deploy.py

```diff
@@ -36,13 +46,45 @@ def run_push(cls, connection, app, env):
 
         cls.run_remote_command(connection, cmd)
 
+    @classmethod
+    def clean_up_old_releases(cls, hostname, connection):
+        current_release = int(cls.get_remote_output(connectoin, 'readlink {}current'.format(cls.DEPLOY_DIR)))
+        current_release_str = str(current_release)
+        logger.debug('{}: current_release = {}'.format(hostname, current_release))
+
+        deployed_releases = cls.get_remote_output(connection, 'ls {}'.format(cls.DEPLOY_DIR)).split()
+        logger.debug('{}: deployed_releases = {}'.format(hostname, deployed_releases))
+
+        for f in deployed_releases:
+            if f == 'current' or f == current_release_str or f.startswith('.'):
+                continue
+            logger.debug('{}: removing deployed release {}'.format(hostname, f))
+            cls.run_remote_command(connection, 'rm -rf {}{}'.format(cls.DEPLOY_DIR, f))
```

Comment:
We should really come up with better solution for this since it is a major security hole